iirfilter-node/index.html

```diff
@@ -1,350 +1,416 @@
-canvasContainer.appendChild(canvasEl);
-// set 2d context and set dimesions
-const canvasCtx = canvasEl.getContext('2d');
-const width = canvasContainer.offsetWidth;
-const height = canvasContainer.offsetHeight;
-canvasEl.width = width;
-canvasEl.height = height;
-
-// set fill and create axis
-canvasCtx.fillStyle = 'white';
-canvasCtx.fillRect(0, 0, width, height);
-
-const spacing = width/16;
-const fontSize = Math.floor(spacing/1.5);
-canvasCtx.lineWidth = 2;
-canvasCtx.strokeStyle = 'grey';
-
-// draw our axis
-canvasCtx.beginPath();
-canvasCtx.moveTo(spacing, spacing);
-canvasCtx.lineTo(spacing, height-spacing);
-canvasCtx.lineTo(width-spacing, height-spacing);
-canvasCtx.stroke();
-// axis is gain by frequency
-canvasCtx.font = fontSize+'px sans-serif';
-canvasCtx.fillStyle = 'grey';
-canvasCtx.fillText('1', spacing-fontSize, spacing+fontSize);
-canvasCtx.fillText('g', spacing-fontSize, (height-spacing+fontSize)/2);
-canvasCtx.fillText('0', spacing-fontSize, height-spacing+fontSize);
-canvasCtx.fillText('Hz', width/2, height-spacing+fontSize);
-canvasCtx.fillText('20k', width-spacing, height-spacing+fontSize);
-
-// dom elements needed
-const loadingSection = document.querySelector('.loading');
-const playButton = document.querySelector('.button-play');
-const filterButton = document.querySelector('.button-filter');
-
-// arrays for our frequency response
-const totalArrayItems = 30;
-// Here we want to create an array of frequency values that we would like to get data about. We could go for a linear approach, but it's far better when working with frequencies to take a log approach, so let's fill our array with frequencies that get larger as array item goes up.
-let myFrequencyArray = new Float32Array(totalArrayItems);
-myFrequencyArray = myFrequencyArray.map(function(item, index) {
-    return Math.pow(1.4, index);
-});
-// We need to create arrays that return the data, these need to be the same size as the origianl frequency array
-let magResponseOutput = new Float32Array(totalArrayItems);
-let phaseResponseOutput = new Float32Array(totalArrayItems);
-
-// let the magic happen! When the file has loaded...
-setupSample()
-	.then((sample) => {
-
-	// remove loading screen
-	loadingSection.style.display = 'none';
-
-	// create out iir filter
-	const iirfilter = audioCtx.createIIRFilter(feedForward, feedBack);
-
-	let srcNode;
-	// play/pause our track
-	playButton.addEventListener('click', function() {
-		if (this.dataset.playing === 'false') {
-
-			// check if context is in suspended state (autoplay policy)
-      if (audioCtx.state === 'suspended') {
-        audioCtx.resume();
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Web Audio API examples: IIR Filter</title>
+    <meta name="description" content="IIR Filter Demo for Web Audio API" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <style type="text/css">
```

Comment:
```suggestion
    <style>
```
Fix Nu HTML checker warning.